AsyncDisplayKit/ASTextNode.mm

```diff
@@ -321,9 +353,14 @@ - (void)calculatedLayoutDidChange
   
   if (layout != nil) {
     ASDN::MutexLocker l(__instanceLock__);
-    if (CGSizeEqualToSize(_constrainedSize, layout.size) == NO) {
-      _constrainedSize = layout.size;
-      _renderer.constrainedSize = layout.size;
+    CGSize layoutSize = layout.size;
+    //textContainerInset
+    layoutSize.width -= _textContainerInset.left + _textContainerInset.right;
+    layoutSize.height -= _textContainerInset.top + _textContainerInset.bottom;
```

Comment:
Any opinions adding `()` to `_textContainerInset.left + _textContainerInset.right` just to make it more readable / clear ...